pkg/skaffold/debug/transform_go.go

```diff
@@ -120,6 +121,13 @@ func (t dlvTransformer) Apply(adapter types.ContainerAdapter, config ImageConfig
 	if spec == nil {
 		newSpec := newDlvSpec(uint16(portAlloc(defaultDlvPort)))
 		spec = &newSpec
+
+		if dmd != nil {
+			spec.wait = dmd.ShouldWait
+		} else {
```

Comment:
by default bools are false - is this set somewhere else? Otherwise, I think you could just delete the else clause.

Reply:
in go sure bools are instantiated as false, but [here](https://github.com/pkoutsovasilis/skaffold/blob/a1bd2365ce9a6f40d92e44dea3935816ea8001fa/pkg/skaffold/debug/transform_go.go#L59) not this one, again if you prefer this boolean to start with a false value I need to rename the `wait` to `continue`